Makefile

```diff
@@ -183,7 +183,15 @@ setup:
 # This target is used to update the OpenAPI docs for use in the sphinx docs.
 # Lumigator must be running on localhost
 update-openapi-docs:
-	./scripts/update_openapi_docs.sh
+	cd lumigator/backend/; \
+	S3_BUCKET=lumigator-storage \
+	RAY_HEAD_NODE_HOST=localhost \
+	RAY_DASHBOARD_PORT=8265 \
+	SQLALCHEMY_DATABASE_URL=sqlite:////tmp/local.db \
+	MLFLOW_TRACKING_URI=http://localhost:8001 \
+	PYTHONPATH=../jobs:$$PYTHONPATH \
+	LUMIGATOR_SECRET_KEY=7yz2E+qwV3TCg4xHTlvXcYIO3PdifFkd1urv2F/u/5o= \
+	python -m backend.openapi_spec
```

Comment:
Is it necessary to have all these env vars? And I think you may have forgotten to commit the script 😅

Reply:
Currently yes. AFAICT the app needs to be live (although not serving traffic) so we need to provide sensible values for the vars.

About the script..... 🤦🏼 sorry!!!!!!